lib/sanity-util.js

```diff
@@ -17,13 +17,19 @@ function convertBlockToHTML({ blocks, cache, entriesById, options, visitedIds })
         return blockToHTML.h('img', { src: normalizedImage.url });
     };
 
+    const allSerializers = {
+      types: {
+          ...options.serializers.types,
+          image: imageSerializer
+      },
+      marks: {
+          ...options.serializers.marks
+      }
+    }
```

Comment:
@rudietuesdays as noted before and although you have added the `serializers` option to the `module.exports.options` in `index.js`, Sourcebit will not automatically add `serializers` key to `options`.

This means that if user runs Sourcebit without providing these options, these lines will throw an exception because serializers will be undefined:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'types')
```

I suggest getting the options in a safely manner like `...(options.serializers && options.serializers.types)`.

If you want to know more about how Sourcebit merges options here is the code in Sourcebit that loads plugin options from config and merges it with `default` options when these are defined.
https://github.com/stackbit/sourcebit/blob/1dde98badc428a87d706bcee0e4c41ffe4eed29e/lib/sourcebit.js#L127-L151